kbase-extension/static/kbase/js/widgets/narrative_core/upload/fileUploadWidget.js

```diff
@@ -85,20 +85,23 @@ define([
                     $dropzoneElem.find('#global-info').css({'display': 'inline'});
                     $dropzoneElem.find('#upload-message').text(this.makeUploadMessage());
 
+                    // If there is a button already in the area, it has to be removed,
+                    // and appened to the new document when additional errored files are added.
+                    if ($dropzoneElem.find('#clear-all-btn').length){
+                        this.deleteClearAllButton();
+                        $dropzoneElem.append(this.makeClearAllButton());
+                    }
+
                 })
                 .on('success', (file, serverResponse) => {
+                    var $successElem = $(file.previewElement);
+                    $successElem.find('#upload_progress_and_cancel').hide();
+                    $successElem.find('#dz_file_row_1').css({"display": "flex", "align-items": "center"});
```

Comment:
Had to use `display: flex` to get the items in the row vertically aligned. I'm wasn't able to do this with any bootstrap trickery.